libshortfin/CMakeLists.txt

```diff
@@ -23,21 +23,14 @@ set(CMAKE_CXX_STANDARD 20)
 # https://discourse.cmake.org/t/cmake-3-28-cmake-cxx-compiler-clang-scan-deps-notfound-not-found/9244/3
 set(CMAKE_CXX_SCAN_FOR_MODULES 0)
 
+# Problems with libfmt without fPIC
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fPIC")
```

Comment:
Just do (vs munging c flags): set(CMAKE_POSITION_INDEPENDENT_CODE ON)

We will probably want to not do this on Windows.